### DISCUSSION TITLE
I get an error after entering console.bat

### DISCUSSION THREAD

**Discussion by CzytelnyNick** [rapid7/metasploit-framework#14982] [Q&A] [answered]
[*] Starting Metasploit Console...
Traceback (most recent call last):
        25: from F:/metasploit/apps/pro/engine/msfpro:17:in `<main>'
        24: from F:/metasploit/apps/pro/engine/lib/metasploit/pro/engine/command/base.rb:66:in `start'
        23: from F:/metasploit/apps/pro/engine/lib/metasploit/pro/engine/command/console.rb:19:in `start'
        22: from F:/metasploit/apps/pro/engine/lib/metasploit/pro/engine/command/console.rb:38:in `start_local'
        21: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/metasploit-framework-6.0.35/lib/metasploit/framework/command/base.rb:82:in `start'
        20: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/metasploit-framework-6.0.35/lib/metasploit/framework/command/console.rb:48:in `start'
        19: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/metasploit-framework-6.0.35/lib/metasploit/framework/command/console.rb:60:in `driver'
        18: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/metasploit-framework-6.0.35/lib/metasploit/framework/command/console.rb:60:in `new'
        17: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/metasploit-framework-6.0.35/lib/msf/ui/console/driver.rb:66:in `initialize'
        16: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/metasploit-framework-6.0.35/lib/msf/ui/console/driver.rb:656:in `choose_readline'
        15: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.5/lib/active_support/dependencies.rb:291:in `require'
        14: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.5/lib/active_support/dependencies.rb:257:in `load_dependency'
        13: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.5/lib/active_support/dependencies.rb:291:in `block in require'
        12: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
        11: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
        10: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
         9: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/rb-readline-0.5.5/lib/readline.rb:8:in `<top (required)>'
         8: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/rb-readline-0.5.5/lib/readline.rb:10:in `<module:Readline>'
         7: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.5/lib/active_support/dependencies.rb:291:in `require'
         6: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.5/lib/active_support/dependencies.rb:257:in `load_dependency'
         5: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.5/lib/active_support/dependencies.rb:291:in `block in require'
         4: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
         3: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
         2: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
         1: from F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/rb-readline-0.5.5/lib/rbreadline.rb:17:in `<top (required)>'
F:/metasploit/apps/pro/vendor/bundle/ruby/2.7.0/gems/rb-readline-0.5.5/lib/rbreadline.rb:1097:in `<module:RbReadline>': HOME environment variable (or HOMEDRIVE and HOMEPATH) must be set and point to a directory (RuntimeError)

Please help me, I don't know what this error is about

**Comment by ccondon-r7** [comment, marked as answer]
Hi! This looks like an issue, not a discussion. You can open an issue here: https://github.com/rapid7/metasploit-framework/issues
